Documentation/manual-install.md

```diff
@@ -38,6 +38,8 @@ $ export METERING_NAMESPACE=metering-$USER
 $ ./hack/openshift-install.sh
 ```
 
+**Note** The default `MeteringConfig` CR has no storage setup so you may want to choose HDFS, do so with: ```export METERING_CR_FILE=manifests/metering-config/hdfs-storage.yaml```, you *must* choose some storage option for a successful install.
```

Comment:
Two things:
- We do not want to recommend HDFS to users.
- This is covered under `## Customize installation`. It might make sense to move that section up now that the minimal configuration requires storage, and update to to mention the configuring storage documentation.